ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-commands.sh

```diff
@@ -0,0 +1,395 @@
+          "enabled" : true,
+          "redirectUris" : [ "http://localhost:8080" ],
+          "webOrigins" : [ "http://localhost:8080" ],
+          "standardFlowEnabled" : true,
+          "directAccessGrantsEnabled" : true,
+          "publicClient" : true,
+          "frontchannelLogout" : true,
+          "protocol" : "openid-connect",
+          "attributes" : {
+            "oidc.ciba.grant.enabled" : "false",
+            "backchannel.logout.session.required" : "true",
+            "oauth2.device.authorization.grant.enabled" : "false",
+            "backchannel.logout.revoke.offline.tokens" : "false",
+            "client.session.idle.timeout" : "7200"
+          }
+        }
+      client-console-test.json: |
+        {
+          "clientId" : "console-test",
+          "enabled" : true,
+          "secret" : "${CONSOLE_CLIENT_SECRET_VALUE}",
+          "redirectUris" : [ "${CLUSTER_CONSOLE}/auth/callback" ],
+          "webOrigins" : [ "${CLUSTER_CONSOLE}" ],
+          "standardFlowEnabled" : true,
+          "directAccessGrantsEnabled" : true,
+          "publicClient" : false,
+          "frontchannelLogout" : true,
+          "protocol" : "openid-connect",
+          "attributes" : {
+            "oidc.ciba.grant.enabled" : "false",
+            "backchannel.logout.session.required" : "true",
+            "oauth2.device.authorization.grant.enabled" : "false",
+            "backchannel.logout.revoke.offline.tokens" : "false",
+            "client.session.idle.timeout" : "7200"
+          }
+        }
+      groupmapper-for-clients.json: |
+        {
+          "name" : "groupmapper",
+          "protocol" : "openid-connect",
+          "protocolMapper" : "oidc-group-membership-mapper",
+          "consentRequired" : false,
+          "config" : {
+            "full.path" : "false",
+            "userinfo.token.claim" : "true",
+            "id.token.claim" : "true",
+            "access.token.claim" : "false",
+            "claim.name" : "groups"
+          }
+        }
+      testusers: |
+        ${TEST_USERS}
+
+keycloak:
+  extraEnv:
+    - name: KC_PROXY
+      value: "edge"
+    - name: KC_HTTP_PORT
+      value: "8080"
+    - name: KC_LOG_LEVEL
+      value: "INFO"
+  http:
+    enabled: true
+  https:
+    enabled: true
+  hostname:
+    hostname: ${KEYCLOAK_HOST}
+    strict: false
+  jvm:
+    memory:
+      initial: 512m
+      max: 1024m
+
+ingress:
+  enabled: true
+  hostname: ${KEYCLOAK_HOST}
+  ingressClassName: nginx 
+  path: /
+  pathType: Prefix
+  tls: true
+  tlsSecret: ${KEYCLOAK_HOST}-tls
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: HTTP
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+
+postgresql:
+  image:
+    registry: docker.io
+    repository: bitnamilegacy/postgresql
+    tag: 17.6.0-debian-12-r0
+
+resources:
+  requests:
+    cpu: 500m
+    memory: 1Gi
+  limits:
+    cpu: 1000m
+    memory: 2Gi
```

Comment:
Same here for any inline rational comments? The resource requirment seems high which can easily cause weird isssue for cluster components or keycloak running.